ansible/roles/products/common/docker-container/tasks/main.yml

```diff
@@ -4,6 +4,16 @@
   register: id
   changed_when: id.rc != 0
 
+- name: set docker_version to be just docker if nvidia_docker is 0
+  when: nvidia_docker=="0"
```

Comment:
I think that nvidia-docker2 is using the `docker` command to run the containers

Reply:
Fixed in the latest commit